pages/api/player/[id]/confirm.ts

```diff
@@ -33,10 +34,27 @@ export default async function handler(
   if (req.method === "PATCH") {
     const playerId = req.query.id as string;
     const { confirmed } = JSON.parse(req.body);
+
+    const player = await prisma.player.findFirst({
+      where: {
+        id: playerId
+      },
+      include: {
+        user: true,
+        tournament: true
+      }
+    });
+
     const result = await prisma.player.update({
       where: { id: playerId },
       data: { confirmed }
     });
+    sendEmail(
+      "surma@salamurhaajat.net",
+      player.user.email,
+      `Ilmoittautumisesi on nyt vahvistettu!`,
+      `Tervetuloa mukaan Helsingin Yliopiston Salamurhaajien turnaukseen ${player.tournament.name}! Tuomaristo on tarkistanut ilmoittautumisesi ja vahvistanut sen.\n\nTämä on automaattinen vahvistusviesti. Älä vastaa tähän viestiin, vaan lähetä viestisi osoitteeseen tuomaristo@salamurhaajat.net`
```

Comment:
`Tervetuloa mukaan Helsingin Yliopiston Salamurhaajien turnaukseen ${player.tournament.name}!\n\nTämä on automaattinen vahvistusviesti. Älä vastaa tähän viestiin, vaan lähetä viestisi osoitteeseen tuomaristo@salamurhaajat.net`

Reply:
Tämmöinen ehdotus, kun tuntuu turhalta toistolta.

Reply:
Kyllä mä sanoisin, että toiston uhallakin on parempi ilmaista mahdollisimman selkeästi ja eksplisiittisesti, mikä vahvistusviesti on kyseessä.

Reply:
Lisäksi tuo lause myös kertoo, 1) kuka on vahvistanut, 2) että ilmoittautuminen on myös tarkistettu. Nääkin on hyvä sanoa ääneen, ettei pelaajalle jää mitään epämääräisyyden tunnetta asiasta.

Reply:
Suggestoinpa tähänkin nyt ton vaihtoehtoisen do not reply -lopetuksen.
```suggestion
      `Tervetuloa mukaan Helsingin Yliopiston Salamurhaajien turnaukseen ${player.tournament.name}! Tuomaristo on tarkistanut ilmoittautumisesi ja vahvistanut sen.\n\nTämä on automaattinen vahvistusviesti. Älä vastaa tähän viestiin. Tuomaristo vastaa peliin liittyviin viesteihin osoitteessa tuomaristo@salamurhaajat.net.`
```

Reply:
Mä sanoisin kyllä myös että mennään alkuperäisellä tekstillä + vaihtoehtoinen do not reply. En usko, että pelaajia pieni toisto niin paljoa haittaa ja varsinkin jos mukana on ensikertalaisia niin on hyvä et viestissä lukee mahdollisimman selkeästi, että ilmoittautumisen suhteen kaikki on nyt kunnossa.